website/docs/sdk-and-devtool/cli-tool.mdx

```diff
@@ -0,0 +1,269 @@
+---
+id: cli-tools
+title: CLI Tools
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Tooltip from "@components/Tooltip";
+
+# CLI Tools
```

Comment:
Should we separate CKB-CLI and OffCKB to different pages (Just like what we do for the SDKs)?

Reply:
I'll give it a thought. let's leave them under cli for now.